diff.py

```diff
@@ -352,11 +375,14 @@ def create_project_settings(settings: Dict[str, Any]) -> ProjectSettings:
         objdump_executable=get_objdump_executable(settings.get("objdump_executable")),
         map_format=settings.get("map_format", "gnu"),
         mw_build_dir=settings.get("mw_build_dir", "build/"),
+        browser=settings.get("browser"),
```

Comment:
I don't think it's right to have this in project settings, it's a personal preference. Use an env var instead. There's also `webbrowser.open` in Python which could serve as a default, though I'm not sure how well it works with WSL or if it's worth the complexity to support (also makes it impossible to give an error message about what kind of env var to create)

Reply:
For some reason `webbrowser.open` run from the python3 prompt in WSL does open me a firefox tab, neat. Even more impressive (imo), `webbrowser.open('client.html')` can open client.html from the wsl filesystem (`file://///wsl$/Ubuntu-18.04/home/dragorn421/asm-differ/client.html`)!
Though as discussed, the open url will eventually be "http://localhost:8000" or smth like that

So I used both an env var and webbrowser.open (env var preferred)